### PR TITLE
docs(core): Initialize browser example docs with updated cdn

### PR DESCRIPTION
**Pull request by uvmetal**
This PR adds an index and an array of embedded references and jsfiddle examples built against the official neon-js@4.5.2 cdn package on jsdelivr.net. It is an effort to improve the quality of documentation as a way to show clear, useful, interactive examples with hopes of accelerating understanding and adoption.

This documentation is a work in progress. This commit is the initial effort to establish formatting and other conventions. The documentation should be reviewed and improved upon over time. Bindings against versions of neon-js <-> Neo API mappings will evolve.

The current fiddles have been tested and confirmed to work. Please comment with any errors, suggestions, or issues.

The documentation must be in master branch in the given format for the fiddles to be tested. As such, all testing was committed against my fork of neon-js coz official and then submitted here.

Thanks!

### CODE REVIEW

File: examples/browser/getblocksysfee/demo/demo.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Neon.settings.httpsOnly = true;
+
+function InvokeOperation()
+{
+  clearHtml();
+
+  // Get an RPC Endpoint (Neo Node)
+  provider.getRPCEndpoint().then(nodeUrl => {
+    const client = Neon.default.create.rpcClient(nodeUrl);
+    // change 'hash' below to 'index' to use the index version of getblock
```
Comment (snowypowers) [inline examples/browser/getblocksysfee/demo/demo.js]: ?

Comment (phetter) [reply examples/browser/getblocksysfee/demo/demo.js]: I'm sorry. What's the question?

Comment (snowypowers) [reply examples/browser/getblocksysfee/demo/demo.js]: I dont see `hash` anywhere below. Maybe its swapped around and meant to change `index` to `hash` to use `hash` version?

Comment (uvmetal) [reply examples/browser/getblocksysfee/demo/demo.js]: Oh, yeah, right, that was a comment left over from the template I was using. It's an error. I've removed the line. Good eyes as always snowy. Thanks for your help!

**Comment by hal0x2328** [comment]
1000 points uvmetal
200 points snowypowers for review

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/589328c622e101065f6717d61a332f93aac71cc68b665e194ef8c874a7e26669

**Pull request merged**